region/norfair/east/Double Chamber.json

```diff
@@ -1171,6 +1174,40 @@
         {"heatFrames": 300}
       ]
     },
+    {
+      "link": [3, 4],
+      "name": "Ripper I-Frames, Spike Jump",
+      "requires": [
+        "canPrepareForNextRoom",
+        "canUseIFrames",
+        {"enemyDamage": {"enemy": "Ripper 2 (green)", "type": "contact", "hits": 1}},
+        {"or": [
+          "canHorizontalDamageBoost",
+          {"and": [
+            "HiJump",
```

Comment:
I'm not sure how you're meant to do this without falling on the crumbles.  Do you hit the ripper on the right and then again on the left, or activate the ripper by walljumping on the right wall and then blind jump to where it will be on the left?
Wither way doesn't use hijump.

Reply:
Hit the Ripper, fall to the left of the crumbles, get a running jump off the spikes and wall jump on the top left ledge. 
Only the HiJump variant can land right of the crumbles, I couldnt get the spring ball variant to jump far enough left

Reply:
That feels fine to do, I just didn't think it would reach.
With springball you want to morph faster for some springfling, or run on the crumbles a bit.  And that is going to be harder than damage boosting on the bug.  And if you want to overcomplicate the strat, you can bounce on the crumbles before touching the ripper to be on the left side with iframes.